src/Controllers/CreateNewEntryController.php

```diff
@@ -3,41 +3,54 @@
 namespace App\Controllers;
 
 use App\Abstracts\Controller;
-
+use App\Interfaces\BlogModelInstanceInterface;
+use App\Validators\Validators;
 use Psr\Http\Message\ResponseInterface as Response;
 use Psr\Http\Message\ServerRequestInterface as Request;
+use Valitron\Validator;
 
-
-class CreateNewEntryController extends Controller
+class CreateNewEntryController extends Controller implements BlogModelInstanceInterface
 {
-    private $BlogModel;
+    private BlogModelInstanceInterface $blogModel;
 
     /**
-     * CreateNewEntryController constructor.
-     * @param $BlogModel
+     * editEntryController constructor.
+     * @param $blogModel
      */
-    public function __construct($BlogModel)
+    public function __construct(BlogModelInstanceInterface $blogModel)
     {
-        $this->BlogModel = $BlogModel;
+        $this->blogModel = $blogModel;
     }
 
-    public function __invoke(Request $request, Response $response, array $args)
+    public function __invoke(Request $request, Response $response, array $args): Response
     {
-        $ResponseData =
+        $responseData =
             [
                 'success' => false,
                 'message' => '',
                 'data' => []
             ];
 
-        $AddNewEntry = $request->getParsedBody()['NewBlogEntry'] ?? null;
+        $newBlogPost = $request->getParsedBody();
+        $blogPost = new Validator($newBlogPost);
 
-        if (!$AddNewEntry)
-        {
-            $ResponseData['message'] = "Please fill all fields";
+        if (!Validators::ValidateNewPost($blogPost)) {
+            $responseData['success'];

```

Comment:
this line does nothing.

Reply:
to explain, this is essentially the save as writting `$blogPost;`

Reply:
I thought it would send the default value of False if you're referring to  line 38